pages/options/options.html

```
@@ -113,6 +113,21 @@
             ></span>
             <span class="label">使用指南</span>
           </button>
+          <button
+            id="tab-about"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)